.github/cspell.json

```diff
@@ -3,43 +3,186 @@
   "language": "en",
   "flagWords": [],
   "words": [
```

Comment:
Note also: these words should be added into the file `.github/.cspell/flame_dictionary.txt`.